app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +12,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.CactusNotes">
+        <activity android:name=".NoteListActivity"></activity>
```

Comment:
This could be a self-closing tag like: 

`<activity android:name=".NoteListActivity"/>`